### ISSUE TITLE
setMaxConnections vs. maintainScriptOrder

### ISSUE THREAD

**Issue by inear** [CreateJS/PreloadJS#61]
When using setMaxConnections with more than 1, it sometimes prevent the the loading queue to finish completely. It stops on the last item and it occurs when the items is received in a different order than the order in the queue, due to the max-connections and the browsers own limitation that might delay the request. Not sure what is causing this, but it's solved by manually setting maintainScriptOrder to false. In my case it's just json-files so I don't need maintainScriptOrder to be true. Thinking it can be good to know for others that experience problems getting the loadingComplete event occasionally. The loading should at least warn or something if setMaxConnections is used and if this is an expected behaviour when maintainScriptOrder is true. Or a more robust check for items still in the queue, so that _loadNext and _checkScriptLoadOrder is fired correctly when items is received in different order than loaded.

```javascript
var queue = new createjs.LoadQueue(true);
queue.setMaxConnections(6);
queue.maintainScriptOrder = false;
```

**Comment by lannymcnie** [comment]
I am unable to reproduce this. Can you provide a sample showing the issue?

**Comment by msgopinath** [comment]
Still i can reproduce if i set max connection  > 1 handleComplete triggered before the loading was completed..Its bcz of preloader loads two times the same queue due to that complete event triggered before the loading is done..

**Comment by digitaloranges** [comment]
I realised what my issue was on the end - I hadn't set some manifest item types to 'spritesheet'.  After I set all my atlases/spritesheets to that, the issue went away

**Comment by lannymcnie** [comment]
There was a bug where an error with images didn't properly surface, causing the queue to stall. This has been fixed in NEXT, and will be available in 0.6.2.
